cumulusci/core/flowrunner.py

```diff
@@ -133,11 +133,8 @@ class FlowCallback(object):
 
      This is how you can carry a reference to the Django model a flow run is related to in MetaCI or MetaDeploy."""
 
-    def __init__(self, ctx=None, _factory=None, *args, **kwargs):
+    def __init__(self, ctx=None):
         self.context = ctx
-        if ctx is None:
-            if callable(_factory):
-                self.context = _factory(*args, **kwargs)
```

Comment:
how'd i know you were gonna delete this? ;) fine fine

Reply:
Not opposed to it in principle, but I get annoyed when I have to write test coverage for code that isn't used anywhere else.

Reply:
i predict something like it will get used in the metadeploy subclass, but i don't have the exact shape yet, this was definitely just a guess.